yaksh/templates/yaksh/course_detail.html

```diff
@@ -75,7 +153,46 @@
             <div class="row">
                 <div class="col-md-4" style="padding-top:10px">
                 <input type="checkbox" name="check" value="{{ enrolled.id }}">&nbsp;
-                <a href="#" data-toggle="tooltip" title=" Full Name: {{ enrolled.get_full_name }}&#013;Institute: {{ enrolled.profile.institute }}&#013;Department: {{ enrolled.profile.department}}&#013;Roll Number: {{ enrolled.profile.roll_number }}">&nbsp;{{ enrolled.username }}</a>
+                <a href="#" data-toggle="modal" data-target="#enroll_detail_{{forloop.counter}}">
+                {{ enrolled.username }}</a>
+                    <div class="modal fade" id="enroll_detail_{{forloop.counter}}" role="dialog">
+                    <div class="modal-dialog">
+                    <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                        <h4 class="modal-title">Information for {{ enrolled.username }} </h4>
+                    </div>
+                    <div class="modal-body">
+                        <table>
+                            <tr>
+                                <td><strong>Full Name :</strong></td>
```

Comment:
We can add a js dom function which will create the modal block with different attributes as and when called for.